<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>category_menu</class>
 <widget class="QDialog" name="category_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>812</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>812</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>812</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>812</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>375</width>
     <height>812</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>375</width>
     <height>812</height>
    </size>
   </property>
   <widget class="QLabel" name="category_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>375</width>
      <height>812</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>375</width>
      <height>812</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>375</width>
      <height>812</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/category_page.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="selete_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 12pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	border: 2px solid rgb(0,0,0);
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="go_recordpage">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>59</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="meal_btn">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>150</y>
      <width>40</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="snack_btn">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>150</y>
      <width>64</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="medicine_btn">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>150</y>
      <width>54</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="car_btn">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>150</y>
      <width>40</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="trans_btn">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>250</y>
      <width>88</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="edu_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>56</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="live_btn">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>250</y>
      <width>40</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="travel_btn">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>250</y>
      <width>40</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="daily_btn">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>350</y>
      <width>40</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exer_btn">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>350</y>
      <width>48</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cloth_btn">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>350</y>
      <width>48</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="beauty_btn">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>350</y>
      <width>40</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
